images.yaml

```diff
@@ -613,6 +622,9 @@
 - name: omnition/opencensus-collector
   patterns:
   - pattern: '>= 0.1.11'
+- name: openpolicyagent/conftest
+  patterns:
+  - pattern: '>= v0.25.0'
```

Comment:
0.27.0 exist AFAICT

Reply:
Picked 0.25.0 because it's specified here for starboard v0.11.0  https://github.com/aquasecurity/starboard/blob/v0.11.0/deploy/helm/values.yaml#L304

Reply:
Maybe we can add a PR to update this for them? Like even on master they are using outdated conftest :D